src/include/nfs/ui.rb

```diff
@@ -600,12 +592,15 @@ def HandleEvent(widget)
 
         UI.ChangeWidget(Id(:fstable), :Items, FstabTableItems(@nfs_entries))
       elsif widget == :editbut
+        return EnableDisableButtons() if entryno.nil?
+
         source_entry = @nfs_entries[entryno] || {}
         if !legacy_entry?(source_entry) || edit_legacy?
           edit_entry(source_entry, entryno)
         end
-      elsif widget == :delbut &&
-          Ops.greater_than(Builtins.size(@nfs_entries), 0)
+      elsif widget == :delbut
+        return EnableDisableButtons() if @nfs_entries.empty? || entryno.nil?
```

Comment:
entryno.nil? means that nothing is selected?

Reply:
Yes. It's a corner-case we found when running embedded in the Partitioner right after loading an empty list. The button is not disabled as it should. Fixing the button to be correctly disabled would mean a way more aggressive change in the y2-storage-ng side. So I decided to introduce this low-impact workaround to handle that incorrect button action.